gufe/settings/models.py

```diff
@@ -167,7 +155,7 @@ class Config:
     small_molecule_forcefield: str = "openff-2.1.1"  # other default ideas 'openff-2.0.0', 'gaff-2.11', 'espaloma-0.2.0'
     """Name of the force field to be used for :class:`SmallMoleculeComponent` """
 
-    nonbonded_method = "PME"
+    nonbonded_method: Literal["PME", "NoCutoff"] = "PME"
```

Comment:
These aren't the only options we need to support. 

```suggestion
    nonbonded_method: Literal["CutoffNonPeriodic", "CutoffPeriodic", "Ewald", "LJPME", "NoCutoff", "PME"] = "PME"
```